packages/eslint-remote-tester/src/config/types.ts

```diff
@@ -46,11 +46,11 @@ export interface Config {
      * Function is called with current repository and its location on filesystem.
      */
     eslintConfig:
-        | Linter.FlatConfig
+        | Linter.FlatConfig[]
         | ((options?: {
               repository: string;
               location: string;
-          }) => Linter.FlatConfig | Promise<Linter.FlatConfig>);
+          }) => Linter.FlatConfig[] | Promise<Linter.FlatConfig[]>);
 
     /** Flag used to set CI mode. `process.env.CI` is used when not set. */
     CI: boolean;
```

Comment:
This can be either `FlatConfig` or an array of them:

https://github.com/AriPerkkio/eslint-remote-tester/blob/83a22c9f2cdefb36c676b4e120bc953f55eb6169/packages/eslint-remote-tester/src/engine/worker-task.ts#L222-L224

https://eslint.org/docs/latest/integrate/nodejs-api#-new-eslintoptions

> ```ts
> export namespace ESLint {
> 	interface Options {
>     overrideConfig?: Linter.Config | Linter.Config[] | null | undefined;
> ```